### ISSUE TITLE
Generate sythetic data with custom locations

### ISSUE THREAD

**Issue by messcode** [xzhoulab/SRTsim#7]
First of all, I'd like to express my appreciation for the useful software provided by your project. I've been exploring SRTsim and found it valuable for my work.

I was wondering if it's possible to generate synthetic data by passing custom locations. In my research, I need to use specific user-defined locations for data generation. I noticed that SRTsim currently supports two location layouts, "grid" and "random," in `strtsim_newlocs`. However, I would like to know if there's a way to pass custom user-defined locations and generate synthetic data based on those locations.

Your insights on this matter would be greatly appreciated.

**Comment by jakezhusph** [comment]
> First of all, I'd like to express my appreciation for the useful software provided by your project. I've been exploring SRTsim and found it valuable for my work.
> 
> I was wondering if it's possible to generate synthetic data by passing custom locations. In my research, I need to use specific user-defined locations for data generation. I noticed that SRTsim currently supports two location layouts, "grid" and "random," in `strtsim_newlocs`. However, I would like to know if there's a way to pass custom user-defined locations and generate synthetic data based on those locations.
> 
> Your insights on this matter would be greatly appreciated.


Yes. In the Shinyapp, you would be able to upload the customized location files.

**Comment by messcode** [comment]
Thx! ：）
